### ISSUE TITLE
Check deps. which use old version of Tapable

### ISSUE THREAD

**Issue by c0b41** [rubenspgcavalcante/webpack-chrome-extension-reloader#48]
working great webpack 4 but some deprecated warning show up
![ekran alintisi](https://user-images.githubusercontent.com/2834954/40576363-9937b1b2-60fd-11e8-9bc8-bbd730a40cbc.PNG)

**Comment by rubenspgcavalcante** [comment]
This error must be from other plugin you make use as I don't use [Tapable](https://github.com/webpack/tapable) in this plugin. 👍

**Comment by ClementVidal** [comment]
Using webpack 4.9.1, i confirm, that as soon as you add the ChromeExtensionReloader plugin, you get this warning.
The corollary is also valid, as soon as you remove the ChromeExtensionReloader the warning goes away.

**Comment by rubenspgcavalcante** [comment]
Humm, there must be some tool to check dependencies deeply, so we can find anything is using old verions of Tapable. If there's not I'll try to do some script to read the package.json files inside node_modules 🤔

**Comment by ClementVidal** [comment]
I'm investigating on that.
I'll let you know if i found anything.

**Comment by ClementVidal** [comment]
I was wrong, this warning does not come from dependencies, but from the plugin itself:

The warning seams to come from the file *ChromeExtensionLoad.ts*:
The *.plugin()* call trigger the Warning.
The point is that compiler inherit from *Tapable* object.

```
apply(compiler) {
    if (process.env.NODE_ENV !== "production") {
      compiler.plugin("compilation", comp =>
        comp.plugin(
          "after-optimize-chunk-assets",
          chunks =>
            (comp.assets = {
              ...comp.assets,
              ...this._injector(comp.assets, chunks)
            })
        )
      );

      compiler.plugin("after-emit", (comp, done) =>
        this._triggerer()
          .then(done)
          .catch(done)
      );
    }
  }
```

**Comment by rubenspgcavalcante** [comment]
closed by #50

**Comment by c0b41** [comment]
@ClementVidal thanks

**Comment by rubenspgcavalcante** [comment]
Fix published in the `0.8.1` thanks to @ClementVidal 👍
